packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/LightCurveViewer/LightCurveViewerContainer.js

```diff
@@ -82,6 +83,8 @@ class LightCurveViewerContainer extends Component {
 
   onLoad (rawData) {
     const { onReady } = this.props
+    // TODO: find a cleaner way to get the container node for the viewer.
+    const target = this.viewer.current.wrappedInstance.svgContainer.current
```

Comment:
I'm planning on refactoring the LCV to not be the connected component to the store and instead move that up to the container here (or have another container specific to the store data connection), so that should make the ref cleaner.

Reply:
I've changed this to use a forward ref, which is a lot cleaner.